tests/unit/api/bucketPutNotification.js

```diff
@@ -61,4 +66,16 @@ describe('putBucketNotification API', () => {
             });
         });
     });
+
+    it('should update bucket metadata with empty bucket notification', done => {
+        bucketPutNotification(authInfo, getNotifRequest(true), log, err => {
+            assert.ifError(err);
+            return metadata.getBucket(bucketName, log, (err, bucket) => {
+                assert.ifError(err);
+                const bucketNotifConfig = bucket.getNotificationConfiguration();
+                assert.deepStrictEqual(bucketNotifConfig, {});
+                return done();
```

Comment:
No need to return done, just calling it works.
I had a big spiel about not needing the `assert.ifErrors`'s but will save that for later...

Reply:
not sure how big of a difference it makes, but i removed the returns